src/mono/sample/wasi/http-p2/Program.cs

```diff
@@ -13,6 +13,7 @@ public static class WasiMainWrapper
 {
     public static async Task<int> MainAsync(string[] args)
     {
+        var never = Task.Delay(100_000_000);
```

Comment:
what's the role of `never` in this code?

Reply:
It will create `wasi-clock` handle in far future. We are testing that the app will finish immediately, instead of blocking the runtime exit forever. We capture it into variable to appease compiler warning about un-awaited Task.